src/test/resources/sql/Answer_audit.sql

```diff
@@ -0,0 +1 @@
+create table Answer_audit(id int(11), sortOrder int(11), answerText varchar(255), action varchar(10),dbUser varchar(255), appUser varchar(255), createdTs timestamp)
```

Comment:
From the AuditConfig merge code review: 
Are these lines all the expected results of individual tests?
Could you create a properties file where the key is the test that's executing and the result is the sql above so that everything can move into one file?